core/cas-server-core-services/src/test/java/org/apereo/cas/authentication/DefaultMultifactorTriggerSelectionStrategyTests.java

```diff
@@ -152,7 +153,7 @@ private static HttpServletRequest mockRequest(final String provider) {
 
     private static RegexRegisteredService mockService(final String... providers) {
         final DefaultRegisteredServiceMultifactorPolicy policy = new DefaultRegisteredServiceMultifactorPolicy();
-        policy.setMultifactorAuthenticationProviders(ImmutableSet.copyOf(providers));
+        policy.setMultifactorAuthenticationProviders(new HashSet<>(Arrays.asList(providers)));
```

Comment:
Yup. Looked like we have a few of these `new HashSet<>(Arrays.asList...` patterns. I'll track this under a separate issue.